### ISSUE TITLE
public tag and `electricobjectslives` don't feel like good long term solutions

### ISSUE THREAD

**Issue by chrisdrackett** [spalt/EO1#9]
For one, if someone knows your tag or the `electricobjectslives` tag they will be able to display whatever they want on your display. I've already seen nudity on the `electricobjectslives` tag and while I don't personally care, I'm sure many users would not be happy with those images (or something worse) showing up on their display.

I personally think the ideal solution is something that uses only a users own media, ideally private. For me, being able to specify a tag that I use on my own private media on flickr would be ideal.

**Comment by spalt** [comment]
@chrisdrackett i too have noticed the NSFW stuff, i think the "private tag" is the near-term solution (i.e. give your own uploads a tag no one would ever think to use and then just use the "electricobjectslives" tag as a repository of all previous EO1 uploaded art that you can pick things you like to download and then upload to your stream)

**Comment by chrisdrackett** [comment]
yeah, the problem for me is (outlined in another issue) is that I don't want to put pictures of my kids online anywhere that is publicly accessible, so currently with this app I can't show them on my EO1.

**Comment by davidshayne** [comment]
This app is amazing! Thank you for saving my EO1 One question: I have given a couple of photos a private tag and no matter which option I use, they never show up. I only seem to be able to get the electronicobjectslives tag. Any advice?

**Comment by spalt** [comment]
@davidshayne just to clarify your photos are still public, yet have a unique tag known only to you, correct?  that's what I mean by "private tag".

**Comment by davidshayne** [comment]
Correct. Photos are public and I selected the "Show photos tagged with" option. But it still defaults to electronicobjectslives

**Comment by spalt** [comment]
@davidshayne may i ask your tag?  i just tagged a few of mine as "e01batch3" (https://www.flickr.com/photos/tags/e01batch3) and using the partner app and putting in "e01batch3" i then see them on the device...

**Comment by davidshayne** [comment]
Yup. shaynebraineo1

Funny enough, when I clicked your link, it showed up in my browser, it showed up in my browser no problem. When I swapped in my own tag, nothing. (Same for when I put it in the search on the EO1.) Photos are set to public, safe on Flickr, but I feel like it's a problem on their end.

**Comment by spalt** [comment]
interesting... if you do a search for the tag it should show up.  can you send a link to one of your public photos that is tagged with "shaynebraineo1"?

**Comment by davidshayne** [comment]
The tag indeed works in the search on flickr. 

Here's one of the photos: [https://www.flickr.com/photos/42833148@N00/53445325805/in/photolist-2pqMgmz-2pqMgr4-2pqMgvN-2pqMgyi](url)

**Comment by davidshayne** [comment]
It indeed works if I search for the tag.

Here's a link to one of the pics:
https://www.flickr.com/photos/42833148@N00/53445325805/in/photolist-2pqMgmz-2pqMgr4-2pqMgvN-2pqMgyi

**Comment by spalt** [comment]
@davidshayne yeah, ya got me... flickr is... strange?  If you go https://www.flickr.com/photos/me/tags you can see and organize your tags.  maybe worth removing that tag, then using a link that should show up there that says "Review untagged items" to give specific items tags that way?  If you figure it out let me know!

**Comment by davidshayne** [comment]
Alas, nothing. Tried deleting and re-applying. Tried creating new tags. Funny enough, when I enter a tag I created it does bring up art, but I think it's from the "ElectricObjectsLives" stream. Or something else. I'm not really sure... (Versus when I put in, say, your batch3 tag, which brought up that specific art)

**Comment by DeviantAnomaly** [comment]
is it possible to remove electric objects lives stream from the default sources and only have my photos or photos tagged with listed as sources on the installation? There is NSFW content published to that and I get the same connection issues others have mentioned and it often shifts from tagged playlist to some very NSFW photos that has been added to that EOL stream.

**Comment by spalt** [comment]
> is it possible to remove electric objects lives stream from the default sources and only have my photos or photos tagged with listed as sources on the installation? There is NSFW content published to that and I get the same connection issues others have mentioned and it often shifts from tagged playlist to some very NSFW photos that has been added to that EOL stream.

that was the intention of specifying your own tag.  i..e you tag your photos with a specific tag known only to you, then tell the app to display only photos using that tag.  but it sounds like tagging in Flickr isn't working exactly right for everyone....

**Comment by davidshayne** [comment]
I have been getting "sending failed" lately when I try to upload from Flickr to the partner app and "Failed to send message" errors when I try to update current tag in the partner app. I've tried multiple resets of both my EO1 and my android device. Thoughts?
